utils/context_logging.go

```diff
@@ -27,9 +27,13 @@ func NewLogger(format string) *slog.Logger {
 		}.NewLocalDevHandler(os.Stdout)
 		logger = slog.New(logHandler)
 	case "json":
-		slogOption := slog.HandlerOptions{ReplaceAttr: GCPLoggerAttributeReplacer}
-		jsonHandler := slog.NewJSONHandler(os.Stdout, &slogOption)
-		logger = slog.New(jsonHandler)
+		logger = slog.New(slog.NewJSONHandler(os.Stdout, nil))
+	case "gcp":
+		creds, err := google.FindDefaultCredentials(context.Background())
```

Comment:
I want to try this, the thing is that it does not work in local dev mode (you get credentials but they don't have a project id attached (though they have a quota project id but that would be hacky)
but that's fine because this only makes sense in a gcp setup anyway